.rubocop.yml

```diff
@@ -1,14 +1,22 @@
 ---
 AllCops:
+  NewCops: enable
   Exclude:
     - lib/riemann/tools/*_parser.tab.rb
     - vendor/bundle/**/*
+require:
+  - rubocop-rake
+  - rubocop-rspec
+Gemspec/RequireMFA:
+  Enabled: false
```

Comment:
I would prefer to require the usage of MFA, but according to rubygems.org "(some) owners are not using multi-factor authentication (MFA)."

Reply:
It's @aphyr - when you get a chance, could you enable MFA on RubyGems if that works? Thanks!

Reply:
Done!